lib/associations/belongs-to-many.js

```diff
@@ -506,10 +506,8 @@ BelongsToMany.prototype.injectGetter = function(obj) {
       instances = [instances];
     }
 
-    options = options || {};
-    options.scope = false;
-
-    _.defaults(options, {
+    options = _.assign({}, options, {
+      scope: false,
```

Comment:
Think this should be: `options = _.assign({raw: true}, options, {scope: false})`. `raw: true` is a default in original, new code has it as always set as `true`